### PR TITLE
Suppress IGT compile warnings on Android

### PR DESCRIPTION
**Pull request by renchenglei**
Jira: None.
Test: IGT build passes on Android and no warnings reported.

### CODE REVIEW

**Comment by renchenglei** [comment]
@kalyankondapally @jennycao
This patch is to clean IGT build compile warnings on Android, PTAL.

**Comment by jennycao** [comment]
LGTM

**Comment by jennycao** [comment]
@kalyankondapally  Pls help to review and merge this PR,

**Comment by renchenglei** [comment]
@kalyankondapally @jennycao 
As system is not stable, some case may cause device reboot. The origin tool run_test.sh is based on piglit(python script), it can't launch on Android. So we add one simple tool to help run IGT test. With this tool, we could run BAT and kms test. If system reboots, we could use "-R" to continue test. If some tests always block system, we could use "-x" to exclude these tests. All test results are located at results folder.

**Comment by jennycao** [comment]
@renchenglei 
suggest to change run-on-android.sh to run_tests_android.sh

File: scripts/run-on-android.sh [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+IGT_TEST_ROOT="$ROOT/intel-ci"
+RESULTS="$ROOT/results"
+
+if [ ! -d "$IGT_TEST_ROOT" ]; then
+    echo "Error: could not find tests directory."
+    exit 1
+fi
+
+function print_help {
+        echo "Usage: run-tests.sh [options]"
```
Comment (jennycao) [inline scripts/run-on-android.sh]: update script name

**Comment by renchenglei** [comment]
@jennycao 
Thanks Jenny, changed name from run-on-android.sh to run_tests_android.sh :)

File: scripts/run_test_android.sh [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+                    else
+                        if [[ $test == *@* ]]; then
+                            ./${test%@*} --run-subtest ${test#*@} | tee -a $log
+                        else
+                            ./$test | tee -a $log
+                        fi
+                        last_line="`tail -1 $log`"
+                        if [[ $last_line == *SUCCESS* || $last_line == *SKIP* || $last_line == *FAIL* ]]; then
+                            tail -1 $log >> $result
+                            echo "hehehe"
```
Comment (jennycao) [inline scripts/run_test_android.sh]: print more information log than "hehehe"

**Comment by jennycao** [comment]
LGTM for second patch
@kalyankondapally  Pls help to merge this PR

**Pull request merged**